elasticsearch/Dockerfile.origin

```diff
@@ -25,10 +25,12 @@ ENV ES_PATH_CONF=/etc/elasticsearch/ \
 
 ARG MAVEN_REPO_URL=https://repo1.maven.org/maven2/
 
+RUN rm /etc/yum.repos.d/*
+ADD art.repo /etc/yum.repos.d
```

Comment:
are we sure we would be able to pull from the pulp repos in origin?

Reply:
This is obsolete right now for `UBI8:latest`.